packages/central-server/app/middleware/versionCompatibility.js

```diff
@@ -70,19 +49,14 @@ export const versionCompatibility = (req, res, next) => {
     return;
   }
 
-  const clientTypes = Object.keys(SUPPORTED_CLIENT_VERSIONS);
-  if (!clientTypes.includes(clientType)) {
-    next(
-      new InvalidClientHeadersError(
-        `The only supported X-Tamanu-Client values are ${clientTypes.join(', ')}`,
-      ),
-    );
+  const clientInfo = VERSION_CONTROLLED_CLIENTS[clientType];
+  if (!clientInfo) {
+    // a non version controlled client; ignore version checking
+    next();
```

Comment:
I think is more in the spirit of what this check is doing. There are only specific clients where we apply version controlling to ensure we're not accidentally connecting to the wrong version. These are just internal really, facility/mobile. So we shouldn't be doing version checking on other clients.

Reply:
Yeah agreed, beautiful

Reply:
Really like that variable name too